src/kernel/include/shared/cos_config.h

```diff
@@ -10,7 +10,7 @@
 
 // After how many seconds should schedulers print out their information?
 #define SCHED_PRINTOUT_PERIOD  30
-//#define COMPONENT_ASSERTIONS   1 // activate assertions in components?
+#define COMPONENT_ASSERTIONS   1 // activate assertions in components?
```

Comment:
Oh, do we not want to commit this? I thought it was a mistake that someone had commented out assertions.